client/widgets/CArtifactHolder.cpp

```diff
@@ -686,7 +672,7 @@ void CArtifactsOfHero::realizeCurrentTransaction()
 								ArtifactLocation(commonInfo->dst.AOH->curHero, commonInfo->dst.slotID));
 }
 
-void CArtifactsOfHero::artifactMoved(const ArtifactLocation & src, const ArtifactLocation & dst)
+void CArtifactsOfHero::artifactMoved(const ArtifactLocation & src, const ArtifactLocation & dst, bool withUpdate)
```

Comment:
withUIUpdate or withRedraw

Reply:
> withUIUpdate or withRedraw

in function uses updateParentWindow() rather than safeRedraw(). withUIUpdate is better I think.